CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 8.6.0 - December 20, 2019
+### Bug fixes
+ - Revert of a commit that resulted in a native crash when cycle debug options was invoked, issue was only impacting v8.6.0-beta.1 [#16074](https://github.com/mapbox/mapbox-gl-native/pull/16074)
+ - Fixed a crash caused by an unintentional minification of the `LocalGlyphRasterizer`. The full class definition is now kept when mimifcations is used by end developer [#102](https://github.com/mapbox/mapbox-gl-native-android/pull/102)
+ - Fixed intermittent offline download failures by, offline download must clear batch buffer at load interrupt [#16074](https://github.com/mapbox/mapbox-gl-native/pull/16074)
+ - Bug fix for overscaling lines [#16045](https://github.com/mapbox/mapbox-gl-native/pull/16045)
+ - Avoid UBSAN runtime error by not using signed int type for anchor segment [#16008](https://github.com/mapbox/mapbox-gl-native/pull/16008)
+ - Correct integration of text-size property, only handle positive values conform to the style spec definition [#16080](https://github.com/mapbox/mapbox-gl-native/pull/16080)
```

Comment:
The text-size fix isn't really different from any other bug fix, right? I think that we could formulate in the usual way we describe bug fixes e.g. 

- Fixed error that would result when 0.0 was passed into `text-size`.